com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/DensityVolume.cs

```diff
@@ -89,13 +89,12 @@ public DensityVolumeArtistParameters(Color color, float _meanFreePath, float _an
             m_EditorAdvancedFade = false;
         }
 
-        internal void Update(bool animate, float time)
```

Comment:
Why removing this parameter? Is it because time outside of "always refresh" enabled does not progress? So it's kind of implicit?

Reply:
exactly, camera.time is set to 0 when "always refresh" is unchecked